### ISSUE TITLE
???You might have run out of disk space, or the file might be too large for your OS????

### ISSUE THREAD

**Issue by jiaoting** [boredazfcuk/docker-icloudpd#355]
IOError while writing file to /iCloud/2023/06/29/IMG_4493.HEIC! You might have run out of disk space, or the file might be too large for your OS. Skipping this file...

显示如上问题，可以保证本地空间足够，请问这个是怎么回事呢

**Comment by boredazfcuk** [comment]
I'm also too lazy to use a translation website, so I've no idea what that last line says.

Judging by the error you've received, I'd check that the volume you are downloading to is not full. I'm guessing you've checked that already, but I got this issue when the drives were not mounted into my container correctly and I filled up my root partition. It's the reason I require the .mounted fail-safe file to be created.

I'd also check that you're not mounting the volume as read-only, that your container has permission to write to the volume etc.
